tests/test_filtsmooth/test_gaussfiltsmooth/test_kalman.py

```diff
@@ -21,88 +21,17 @@ def problem():
     return ornstein_uhlenbeck()
 
 
-# Predictions
-
-
-@pytest.fixture
-def predict():
-    """Classical prediction."""
-    return pnfs.predict_via_transition
-
-
-@pytest.fixture
-def predict():
-    """Square-root prediction."""
-    return pnfs.predict_sqrt
-
-
-# Measures
-
-
-@pytest.fixture
-def measure():
-    """Classical measure."""
-    return pnfs.measure_via_transition
-
-
-@pytest.fixture
-def measure():
-    """Square-root measure."""
-    return pnfs.measure_sqrt
-
-
-# Updates
-
-
-@pytest.fixture
-def update():
-    """Classical update."""
-    return pnfs.update_classic
-
-
-@pytest.fixture
-def update():
-    """Square-root update."""
-    return pnfs.update_sqrt
-
-
-@pytest.fixture
-def update():
-    """Iterated classical update."""
-    stopcrit = pnfs.StoppingCriterion()
-    return pnfs.iterate_update(pnfs.update_classic, stopcrit=stopcrit)
-
-
-# Smoothing steps
-
-
-@pytest.fixture
-def smooth_step():
-    """Classical smoothing step."""
-    return pnfs.rts_smooth_step_classic
-
-
-@pytest.fixture
-def smooth_step():
-    """Square-root smoothing step."""
-    return pnfs.rts_smooth_step_sqrt
-
-
 # Test setup
 
 
 @pytest.fixture
-def kalman(problem, predict, measure, update, smooth_step):
+def kalman(problem):
```

Comment:
Ok I have to ask: Do we still test the different implementations here or not? If yes, where in the code do I see that?

Reply:
Not in here, but their implementations are tested in statespace